packages/fxa-auth-server/lib/routes/totp.js

```diff
@@ -140,6 +140,10 @@ module.exports = (log, db, mailer, customs, config) => {
 
         await db.deleteTotpToken(uid);
 
+        // Downgrade the session to email-based verification when TOTP is
+        // removed. #5154
```

Comment:
nit: think it would be worth commenting here explicitly, that we know the session is already verified and so there's no security problem with re-marking it as verified with a different method.